docs/design/datacontracts/RuntimeTypeSystem.md

```diff
@@ -159,6 +159,9 @@ partial interface IRuntimeTypeSystem : IContract
     // Get an instruction pointer that can be called to cause the MethodDesc to be executed
     public virtual TargetCodePointer GetNativeCode(MethodDescHandle methodDesc);
 
+    // Gets the GCCoverageInfo pointer if available.
+    // If not enabled in the target process, returns null.
+    public virtual TargetPointer? GetGCCoverageInfo(MethodDescHandle methodDesc);
```

Comment:
```suggestion
    public virtual bool TryGetGCCoverageInfo(MethodDescHandle methodDesc, [NotNullWhen(true)] out TargetPointer? pointer);
```

I would try to avoid `null` as a sentinel value. Using "Try" semantics are clearer in practice.

Reply:
In this case I prefer using `TryGet` because it allows us to differentiate between not being able to get the field and reading a field with a `nullptr`. 

This isn't required for the GetMethodDescData implementation, but I think it is worth maintaining so we can know if `HAVE_GCCOVER` is enabled in the target.

Reply:
> This isn't required for the GetMethodDescData implementation, but I think it is worth maintaining so we can know if HAVE_GCCOVER is enabled in the target.

Why is that scenario important? If that question is needed, then please create a `IsGCCoverEnabled`.

Reply:
Okay, I'll simplify to Elinor's suggestion.